runestone/activecode/activecode.py

```diff
@@ -229,6 +232,60 @@ def run(self):
         if 'gradebutton' not in self.options:
             self.options['gradebutton'] = ''
 
+        if self.content:
+            if '====' in self.content:
+                idx = self.content.index('====')
+                source = "\n".join(self.content[:idx])
+                suffix = "\n".join(self.content[idx+1:])
+            else:
+                source = "\n".join(self.content)
+                suffix = "\n"
+        else:
+            source = '\n'
+            suffix = '\n'
+        try:
+            engine = create_engine(env.config['dburl'])
+            meta = MetaData()
+            course_name = env.config['course_id']
```

Comment:
``env.config.html_context['course_id']``   Everything from template_args in the pavement.py file gets into there.

However, I'm trying to do a build and even though I added a line to my conf.py with a value for dburl I'm getting an exception that 

```
The exception is  No such config value: dburl
Unable to save to source_code table in activecode.py. Possible problems:
  1. dburl or course_id are not set in conf.py for your book
  2. unable to connect to the database using dburl

This should only affect the grading interface. Everything else should be fine.
```

I added a line to print out the exception.  How did you add the dburl to your conf.py file?